dexter-jenkins/dexter-jenkins/.metadata/.plugins/org.eclipse.core.runtime/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,2 @@
+#Tue May 08 10:53:24 CEST 2018
+org.eclipse.jdt.core.classpathVariable.M2_REPO=C\:\\Users\\v.sanko\\.m2\\repository
```

Comment:
This file contains local path. It should be removed.,

Reply:
Done